src/applications/personalization/dashboard-2/components/claims-and-appeals/Appeal.jsx

```diff
@@ -65,43 +65,41 @@ const Appeal = ({ appeal, name }) => {
   appealTitle = capitalizeFirstLetter(appealTitle);
 
   return (
-    <div className="vads-l-col--12 medium-screen:vads-l-col--6 small-desktop-screen:vads-l-col--8 medium-screen:vads-u-padding-right--3">
-      <div className="vads-u-padding-y--2p5 vads-u-padding-x--2p5 vads-u-background-color--gray-lightest">
-        <h3 className="vads-u-margin-top--0">
-          {appealTitle}
-          {/* Claim for compensation received June 7, 1999 */}
-        </h3>
-        <div className="vads-u-display--flex">
-          <i
-            aria-hidden="true"
-            className={`fas fa-fw fa-check-circle vads-u-margin-right--1 vads-u-margin-top--0p5 vads-u-color--green`}
-          />
-          <div>
+    <div className="vads-u-padding-y--2p5 vads-u-padding-x--2p5 vads-u-background-color--gray-lightest">
+      <h3 className="vads-u-margin-top--0">
+        {appealTitle}
+        {/* Claim for compensation received June 7, 1999 */}
+      </h3>
+      <div className="vads-u-display--flex">
+        <i
+          aria-hidden="true"
+          className={`fas fa-fw fa-check-circle vads-u-margin-right--1 vads-u-margin-top--0p5 vads-u-color--green`}
+        />
+        <div>
+          <p className="vads-u-margin-y--0">
+            Status: {getStatusContents(appeal, name).title}
+          </p>
+          {appeal.attributes.description && (
             <p className="vads-u-margin-y--0">
-              Status: {getStatusContents(appeal, name).title}
+              {appeal.attributes.issues.length === 1 ? 'Issue' : 'Issues'} on
+              {isAppeal ? ' appeal' : ' review'}:{' '}
+              {appeal.attributes.description}
             </p>
-            {appeal.attributes.description && (
-              <p className="vads-u-margin-y--0">
-                {appeal.attributes.issues.length === 1 ? 'Issue' : 'Issues'} on
-                {isAppeal ? ' appeal' : ' review'}:{' '}
-                {appeal.attributes.description}
-              </p>
-            )}
-            {requestEvent && (
-              <p className="vads-u-margin-y--0">
-                Submitted on: {moment(requestEvent.date).format('MMMM D, YYYY')}
-              </p>
-            )}
-          </div>
+          )}
+          {requestEvent && (
+            <p className="vads-u-margin-y--0">
+              Submitted on: {moment(requestEvent.date).format('MMMM D, YYYY')}
+            </p>
+          )}
         </div>
-        <a
-          aria-label={`View details of ${appealTitle} `}
-          className="usa-button-primary"
-          href={`/track-claims/appeals/${appeal.id}/status`}
-        >
-          View details
-        </a>
       </div>
+      <a
+        aria-label={`View details of ${appealTitle} `}
+        className="usa-button-primary"
+        href={`/track-claims/appeals/${appeal.id}/status`}
+      >
+        View details
```

Comment:
Shouldn't this be a `CTALink` according to the design? https://vsateams.invisionapp.com/share/SH10HT8JCKYM#/screens/448032782